src/icon-tab-bar.scss

```diff
@@ -132,10 +132,25 @@ $fd-icon-tab-bar-semantic-values: (
   }
 }
 
-%fd-tabs-focus {
+@mixin fd-tabs-focus($outlineColor: var(--sapContent_FocusColor), $offset: false) {
+  outline-color: $outlineColor;
   outline-style: var(--sapContent_FocusStyle);
   outline-width: var(--sapContent_FocusWidth);
-  outline-color: var(--sapContent_FocusColor);
+
+  @if $offset {
+    outline-offset: 0.0625rem;
+  }
+}
+
+@mixin setNavigationTabColors($color: var(--sapShell_Navigation_Selected_TextColor)) {
```

Comment:
I see that every other mixin named in kebab-case and only this one in camelCase, should it be unified?

Reply:
you are absolutely right. I don't know where my mind was when I did this :D Probably on vacation. Thanks for noticing.Fixing it right away  :)